tests/particle_output_hdf5/particles.xdmf

```diff
@@ -0,0 +1,72 @@
+----------------
+##10      <==             particles/particle-00000.h5:/nodes
+##10      ==>             particles/particles-00000.h5:/nodes
+
+##10      #:1   <== particles/particle-00000.h5
+##10      #:1   ==> particles/particles-00000.h5
```

Comment:
I am not sure where you saw these lines. Maybe this was a bug on github?
I have rebased one more time, but if the clang tests keep timing out I would say lets just merge nevertheless.